src/routes/api.ts

```diff
@@ -0,0 +1,37 @@
+import { FastifyInstance, FastifyReply, FastifyRequest } from 'fastify';
+import ConfigFactory from '../config/config-factory';
+import { Config, ConfigSystem } from '../config';
+
+export default async function api(fastify: FastifyInstance) {
+  fastify.get('/api', (req: FastifyRequest, reply: FastifyReply) => {
```

Comment:
can we remove this? is it adding value? would prefer to version endpoints like `/api/v1/config`